src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/dto/visit/scheduler/PrisonDto.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler
+
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.LocalDate
+
+@Schema(description = "Prison dto")
+data class PrisonDto(
+
+  @Schema(description = "prison code", example = "BHI", required = true)
+  var code: String,
+
+  @Schema(description = "is prison active", example = "true", required = true)
```

Comment:
For the Staff UI we don't need `active` (it should only be dealing with active prisons) and `excludeDates`. Doesn't really matter if they're included I guess but we could theoretically have a slimmer Dto 🤷‍♂️

Reply:
that's true, but you should only be asking for active prison, since you got the list beofore